### ISSUE TITLE
Implement the account manager

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#45]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

An account manager class.

### Why

To separate lower-level ASP code into a manager class.

### Issue expands upon

This issue expands upon another issue(s): #41, #42

### Blockers

This issue is blocked by the following issue(s): #`issue number`
